lark/grammar.py

```diff
@@ -50,11 +52,11 @@ class RuleOptions(Serialize):
     __serialize_fields__ = 'keep_all_tokens', 'expand1', 'priority', 'template_source', 'empty_indices'
 
     def __init__(self, keep_all_tokens=False, expand1=False, priority=None, template_source=None, empty_indices=()):
```

Comment:
Shouldn't the types be attached to the function parameters?

Does it work the same in the official tooling?

Reply:
They should be at both places.

Reply:
I believe MegaIng is right. I will add them in both places.